### PR TITLE
[SchemaRegistry] update serializer dependency to srb3

### PR DESCRIPTION
**Pull request by swathipil**
fixes: #20628

### CODE REVIEW

File: sdk/schemaregistry/azure-schemaregistry-avroserializer/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 - `auto_register_schemas` keyword argument has been added to `AvroSerializer`, which will allow for automatically registering schemas passed in to the `serialize`, when set to `True`, otherwise `False` by default.
 - `value` parameter in `serialize` on `AvroSerializer` takes type `Mapping` rather than `Dict`.
+- Depends on `azure-schemaregistry==1.0.0b3`.
```
Comment (yunhaoling) [inline sdk/schemaregistry/azure-schemaregistry-avroserializer/CHANGELOG.md:9]: (I usually put dependency update on other sections (e.g. bug fixed or other changes))

File: sdk/schemaregistry/azure-schemaregistry-avroserializer/samples/eventhub_send_integration.py [hunk 2]
```diff
@@ -59,10 +59,9 @@ def send_event_data_batch(producer, serializer):
 
 
 # create a AvroSerializer instance
-# TODO: after 'azure-schemaregistry==1.0.0b3' is released, update 'endpoint' to 'fully_qualified_namespace'
```
Comment (yunhaoling) [inline sdk/schemaregistry/azure-schemaregistry-avroserializer/samples/eventhub_send_integration.py:62]: is there other TODOs left in the code base?

Comment (swathipil) [reply sdk/schemaregistry/azure-schemaregistry-avroserializer/samples/eventhub_send_integration.py:62]: no I just looked for them and removed all of them in the tests/samples/code

**Review by yunhaoling — APPROVED** [review]
LGTM!

File: sdk/schemaregistry/azure-schemaregistry-avroserializer/samples/eventhub_send_integration.py (hunk 2 above)
Comment (swathipil) [reply sdk/schemaregistry/azure-schemaregistry-avroserializer/samples/eventhub_send_integration.py:62]: will check one more time after merging

**Pull request merged**